src/main/java/com/google/cloud/spanner/myadapter/parsers/BooleanParser.java

```diff
@@ -16,14 +16,18 @@
 
 import com.google.api.core.InternalApi;
 import com.google.cloud.spanner.ResultSet;
+import com.google.common.collect.ImmutableSet;
+import java.util.Set;
 
 /**
  * Parse specified data to boolean. For most cases it is simply returning the bit representation.
  */
 @InternalApi
 public class BooleanParser extends Parser<Boolean> {
-  private static final byte[] TRUE_VALUE_BYTES = new byte[] {'1'};
+  static final byte[] TRUE_VALUE_BYTES = new byte[] {'1'};
   private static final byte[] FALSE_VALUE_BYTES = new byte[] {'0'};
+  public static final Set<String> TRUE_VALUES = ImmutableSet.of("true", "1");
+  public static final Set<String> FALSE_VALUES = ImmutableSet.of("false", "0");
```

Comment:
Same.

Reply:
This is public as we use it to determine if the received value is true or false even outside this class. For example autocommit can be set by either setting it to 'true' or to '1'.